protonvpn_cli/cli.py

```diff
@@ -435,14 +443,13 @@ def set_protonvpn_tier(write=False, init=False):
             # Check if the choice exists in the dictionary
             protonvpn_plans_selection[user_tier]
             if not init and user_tier == 5:
-                configure_cli()
+                return
```

Comment:
I've replaced all "configure_cli()" calls with return, as to not make the process go deeper and deeper.